### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by Adewunmi97**
-Suggested branch name game_logic.
-Added real logic for the game.

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Game
+  attr_accessor :player1, :player2, :board, :winning_combination
```
Comment (SafaErden) [inline lib/game.rb]: - [ ] The student uses attr_readers/attr_writers. Only uses attr_accessors if they really need it and must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

In this case, the winning_combination shouldn't be updated. So you should use attr_reader for it.

File: bin/main.rb [hunk 2]
```diff
@@ -1,97 +1,137 @@
...
-def free_cell?(board, choice)
-  return false if board[choice - 1] == 'X' || board[choice - 1] == 'O'
-
-  true
+def restart(game, board)
+  space
+  space
+  space
+  puts 'Do You Want To Play Again?'
+  puts 'Enter Y or N'
```
Comment (SafaErden) [inline bin/main.rb:81]: - [ ] The game also ends when a user presses something other than 'N', so please fix the instructions.

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️

**Hi TEAM, your project is awesome. 👐 However, we need a little fix before we can proceed. Kindly find the comments below**
##

**REQUIRED CHANGES**

- [ ] The application should tell if there was a draw after players move, please fix it.

image

- [ ] All game-logic related resources (classes and/or modules) are kept in the `lib` directory. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/lib_dir_exist.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

##

**NOTES**

Friendly reminder: Remember it's important to have a professional repo so you can increase your job opportunities. Please take a step back and read the lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.) Your commit messages contain typos and most of them are not descriptive enough. Please pay more attention when you are committing in your upcoming projects and use imperative tense for them.

##

**SUMMARY**

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

##

*Kindly make the changes and submit another code review once you are ready💪*

Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/) 🗨️

Happy coding_ 👨‍💻 

**Safa ERDEN**

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**